src/sentry/relay/globalconfig.py

```diff
@@ -15,7 +15,6 @@
     "profiling.profile_metrics.unsampled_profiles.platforms",
     "profiling.profile_metrics.unsampled_profiles.sample_rate",
     "profiling.profile_metrics.unsampled_profiles.enabled",
-    "profiling.generic_metrics.functions_ingestion.enabled",
```

Comment:
If we remove this, the flag also won't be forwarded to external relays anymore. Does that mean it can also be removed from Relay (see below)?

https://github.com/getsentry/relay/blob/06b174a7dd417e3edf6a6a0278564c0f5e28c2c5/relay-dynamic-config/src/global.rs#L205-L214

Reply:
Correct, I think it's safe to remove it from Relay as well as we won't to drop it entirely.